bundles/org.openhab.binding.lifx/src/main/resources/OH-INF/thing/channel.xml

```diff
@@ -4,13 +4,38 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
+	<channel-type id="abstemperature" advanced="true">
+		<item-type>Number</item-type>
+		<label>Temperature</label>
+		<description>Sets the color temperature of the light in Kelvin</description>
+		<category>ColorLight</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>ColorTemperature</tag>
+		</tags>
+		<state min="1500" max="9000" step="1" pattern="%.0f K"/>
+	</channel-type>
+
+	<channel-type id="abstemperaturezone" advanced="true">
```

Comment:
Now this is really identical to the one above and should thus be removed.
If all you need to do is to define a different label/desc, this can easily be overwritten for the channel (which you already do in the code).

Reply:
In my latest commit I've reused the system default channel types where possible @kaikreuzer.

Reply:
👍